Core/Services/StitchCoreRemoteMongoDBService/Sources/StitchCoreRemoteMongoDBService/Sync/CoreDocumentSynchronizationConfig.swift

```diff
@@ -238,15 +264,15 @@ internal struct CoreDocumentSynchronization: Hashable {
 
      - parameter atVersion: the version for which the write as completed on
      */
-    mutating func setPendingWritesComplete(atVersion: Document) {
+    mutating func setPendingWritesComplete(atVersion: Document) throws {
         docLock.writeLock()
         defer { docLock.unlock() }
         self.lastUncommittedChangeEvent = nil
```

Comment:
[nit] not your code, but Doug and I spoke with Eric and we decided that we should drive-by rename `lastUncommittedChangeEvent` to `uncommittedChangeEvent` since the "last" is a misnomer. There is only ever one uncommitted change event since they're always coalesced.

Reply:
+1 please fix now